front-end/generate-task-list.js

```diff
@@ -0,0 +1,65 @@
+const addTaskForm = document.querySelector('.task-form');
+const taskList = document.querySelector('.task-list');
+const tasks = [];
+const STATUSES = [{
```

Comment:
This is not what I meant, in the previous comment I pasted the exact code regarding `STATUSES`